app/controllers/api/v1/videos_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V1::VideosController < ApplicationController
+  def index
+    @videos = Video.all 
+    render json: @videos.map { |user| 
+    if user.video.attached? 
+      user.as_json.merge(video_path: url_for(user.video))
+    else 
+      user.as_json only: :title
+    end }
```

Comment:
@moizansari22 Kindly fix the alignment issue and share the response of this endpoint.